dbt_subprojects/daily_spellbook/models/addresses/chains/addresses_ethereum_info.sql

```diff
@@ -0,0 +1,24 @@
+{% set blockchain = 'ethereum' %}
+
+{{
+    config(
+        schema = 'addresses_' + blockchain,
+        alias = 'info',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'delete+insert',
+        unique_key = ['address'],
```

Comment:
Update strategy again:
```suggestion
        incremental_strategy = 'merge',
        unique_key = ['address'],
        merge_update_columns = ['executed_tx_count', 'max_nonce', 'is_smart_contract', 'namespace', 'name', 'last_seen', 'last_tx_block_number'],
```